R/fit_SE.combinations.R

```diff
@@ -231,7 +231,7 @@ fit_SE.combinations <- function(se,
         colnames(complete_subset), 
         c(norm_type, "row_values", "col_values", "codil_values")
       )
-      mat <- as.matrix(complete_subset[, ..keep])
+      mat <- as.matrix(complete_subset[, keep, with = FALSE])
```

Comment:
How about `subset(complete_subset, select = keep)`?

Reply:
For now, we use `[]` for subsetting in all the packages so I wouldn't mix different functions for the same thing